app/models/setting.js

```diff
@@ -50,7 +50,7 @@ export default ModelBase.extend({
   stripeClientId             : attr('string'),
   stripeSecretKey            : attr('string'),
   stripePublishableKey       : attr('string'),
-  stripeTestCliendId         : attr('string'),
```

Comment:
Why this passed in review?

Reply:
@iamareebjamal Typo mistake.

Reply:
> Why this passed in review?

Might the PR be of large size, Thats why we would have missed noticing the error.

Reply:
Why does anything pass in review indeed 😢